src/main/java/duke/Task.java

```diff
@@ -0,0 +1,55 @@
+package main.java.duke;
+
+
+public class Task {
```

Comment:
Instead of having `toFileFormat()` return `null`, the `Task` class can be implemented as an `abstract` class & the `toFileFormat()` as an `abstract` method. 

This wouldn't affect your current implementation with `ToDo`, `Deadline` & `Event` already extending `Task`

```suggestion
public abstract class Task {
```